alacritty/src/cli.rs

```diff
@@ -289,7 +292,10 @@ impl Options {
             config.debug.log_level = max(config.debug.log_level, LevelFilter::Info);
         }
 
-        config
+        match config.validate() {
```

Comment:
A `validate` method doesn't make a lot of sense to me. The config file is valid, the path just isn't accessible, but that doesn't mean the config is invalid. This is also the only case we want to check, so we shouldn't over-complicate things.